scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -0,0 +1,252 @@
+
+    gradle_build_command = [target_android_project_path / f'gradlew{android_support.GRADLE_EXTENSION}', f'assemble{build_config.title()}']
+    has_errors = exp.process_command(gradle_build_command, cwd=target_android_project_path)
+    if has_errors:
+        raise exp.ExportProjectError("Gradle Build Failure.")
+
+    if deploy_to_device:
+        
+        activity_name = f'{ctx.project_name}Activity'
+        logger.info(f"Activity name set to {activity_name}")
+
+        if not org_name:
+            org_name = f'org.o3de.{ctx.project_name}'
+            logger.info(f"Organization name set to  {org_name}")
+
+        signing_config_file_path = target_android_project_path /'app/build.gradle'
+
+        try:
+            with open(signing_config_file_path,'r') as sf:
+                sf_text = sf.read()
+                signing_config_verified = ('signingConfigs' in sf_text and
+                                           'storeFile' in sf_text and
+                                           'storePassword' in sf_text and
+                                           'keyPassword' in sf_text and
+                                           'keyAlias' in sf_text)
+                if signing_config_verified:
+                    android_support.deloy_to_android_device(target_android_project_path,
+                                                build_config,
+                                                android_sdk_home,
+                                                org_name,
+                                                activity_name)
+        except RuntimeError:
+            logger.error("Unable to verify build.gradle file or that it contains proper signing configs. Aborting deployment...")
+        
+
+def export_source_android_parse_args(o3de_context: exp.O3DEScriptExportContext,
+                                     export_config: command_utils.O3DEConfig,
+                                     android_config: command_utils.O3DEConfig):
+    parser = argparse.ArgumentParser(
+                    prog=f'o3de.py export-project -es {__file__}',
+                    description="Exports a project as an Android APK that is optionally deployed to an android device. "
+                                "In order to use this script, the engine and project must be setup and registered beforehand. ",
+                    epilog=exp.CUSTOM_CMAKE_ARG_HELP_EPILOGUE,
+                    formatter_class=argparse.RawTextHelpFormatter,
+                    add_help=False
+    )
+    
+    eutil.create_common_export_params_and_config(parser, export_config)
+
+    parser.add_argument('--deploy-to-android',action='store_true',help='At completion of build, deploy to a connected android device.')
+    
+    parser.add_argument('--org-name', type=str, default=f'org.o3de.{o3de_context.project_name}',
+                        help='The name of the organization identifier (i.e. org.o3de) used to construct the full activity name.')
```

Comment:
I don't think you can set the org-name here. I think this value matches the `namespace` value in the generated build.gradle file for `app`, and it happens to be 'org.o3de.<project_name>' by default, which is why the default value here works. If you change the org here, i think there will be a failure.

Reply:
ok, so it sounds like activity and org are basically set by the project structure before hand. I will go ahead and remove this, and just have the export script calculage the org name based on 'org.o3de.<project_name>'